osidb/urls.py

```diff
@@ -19,7 +19,9 @@
 
 router = routers.DefaultRouter(trailing_slash=False)
 router.register(r"flaws", FlawView)
-router.register(r"flaw_comments", FlawCommentView)
+router.register(
+    r"flaws/(?P<flaw_id>[^/.]+)/comments", FlawCommentView, basename="flawcomments"
```

Comment:
This router works. However, I cannot say it will not cause any errors/warnings.

Btw, when I manually ran the command to generate openapi schema last time, there already were about 50 warnings/errors.

Reply:
Yes, there are additional errors when compared against the `master` commit this branch is based on:

```
Warning #8: FlawCommentView: could not derive type of path parameter "flaw_id" because it is untyped and obtaining queryset from the viewset failed. Consider adding a type to the path (e.g. <int:flaw_id>) or annotating the parameter type with @extend_schema. Defaulting to "string".
Warning #9: FlawCommentView: Failed to obtain model through view's queryset due to raised exception. Prevent this either by setting "queryset = Model.objects.none()" on the view, checking for "getattr(self, "swagger_fake_view", False)" in get_queryset() or by simply using @extend_schema. (Exception: 'flaw_id')
Warning #10: FlawCommentView: could not derive type of path parameter "comment_id" because it is untyped and obtaining queryset from the viewset failed. Consider adding a type to the path (e.g. <int:comment_id>) or annotating the parameter type with @extend_schema. Defaulting to "string".
```

I'll look into it on Monday to try and fix these errors. @Elkasitu @osoukup in case you already did something like that in the past, any tips or pointers, please?

Reply:
I know we have some outstanding warnings in the schema generation and I am not sure all of them can be easily resolve (not too familiar with that). The important is whether the resulting schema looks as we expect. You can check the generated documentation on the `swagger` web page of your local instance and check that there are all expected fields and no more and that they are mandatory or read-only as they should.

Reply:
It's not necessary to clean up the warnings if it's only warnings, my questions was more whether this could introduce some sort of confusion between the two views when resolving the route, which does not seem to be the case.

Reply:
In the end I fixed the new warnings because I was too deep in learning how schema generation works. The new part of the schema was somewhat nonsensical, so it needed a fix anyway.

Fixed in commit 7ff735b65a892a63e3be3f14668e70356743c595.

As for the route resolving, it should work fine. http://127.0.0.1:8000/osidb/api/v1/blablabla shows the generated routes and they seem fine (thanks @jobselko for the tip).